team/pd/team.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
I suggest updating the path to `teams/pd/team.json`

Reply:
updated